test/t8_forest_incomplete/t8_gtest_empty_local_tree.cxx

```diff
@@ -127,22 +134,28 @@ TEST_P (local_tree, test_empty_local_tree)
 {
   /* Number of instances */
   const uint32_t      num_instances = 1 << MPI_size;
-  for (uint32_t instances = 1; instances < num_instances; instances++) {
-    /* Remove all local elements for every process \a removes[rank] == 0 */
-    std::bitset < MAX_NUM_RANKS > removes (instances);
+  for (uint32_t instances = 1; instances < num_instances - 1; instances++) {
```

Comment:
each instance is its own testcase. To improve debugging for future errors it might be helpfull to parametrize the test over this variable. That way, we can pinpoint the failing tests more easily.

Reply:
I did this in the test `t8_gtest_empty_global_tree`. But in this test the number of test cases is defined by the number of MPI ranks, which also defines the level of the uniform forest (the one tree) to ensure that each process contains elements.
I don't know how to parameterize the test using a non-constant variable that is set at runtime.